.github/workflows/deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: Shopify Analytics Deploy
+
+on:
+  push:
+    branches: [ master ]
+  # Allow running this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  ECR_IMAGE_REPO: "884681002735.dkr.ecr.us-east-1.amazonaws.com/shopify-analytics"
+  DEPLOY_IAM_ROLE: "arn:aws:iam::884681002735:role/github_actions/github-actions-role"
```

Comment:
I don't see new policies for this role:
https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-west-1#/roles/details/github-actions-role?section=permissions

Reply:
We also probably need to update the trust relationships to include this repo:
https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-west-1#/roles/details/github-actions-role?section=trust_relationships

Reply:
Both those concerns are addressed in the PR for the IAM terraform: https://github.com/iFixit/server-templates/pull/4260